### ISSUE TITLE
a minor bug in NeuralNet.py

### ISSUE THREAD

**Issue by LilySnow** [DeepRank/deeprank#28]
When we do classification, the following line of NeuralNet.py:
`loss = self.criterion(outputs,targets)
`
needs to be changed to
`loss = self.criterion(outputs,targets.view(-1))`

Otherwise, we will get “multi-target not supported” error.

**Comment by NicoRenaud** [comment]
Thanks. That's fixed now
